### DISCUSSION TITLE
[Projects Beta] Tree view with "Tracked in"

### DISCUSSION THREAD

**Discussion by JeanLuc-Thell** [community/community#6273] [Projects and Issues]
In order to more easily manage big items broken down into small pieces, we have the ability to create Tasks and track them in a parent issue.
If in [Projects Beta] we could have a tree view enabling us to view those tracked issues as children of parent issue, it will be easier to prioritize and follow progress of parent issue.

In the same manner, issues do have a "Tracked in" and "Task" properties enabling us to know how many child issues exists and which is the parent, issue. If we could display them in table and board view that would ease the backlog management

**Comment by justinfagnani** [comment]
This would be very, very useful for visualizing hierarchical issues tracked with task lists. Being able to expand a task list in the table view to make a tree would be huge.

**Comment by TWiStErRob** [comment]
This is an amazing idea! Thanks for raising. Is there any workaround so far?

**Comment by numbsafari** [comment]
Here in 2024, chiming in on how helpful this would be.

+1 👍 🚀
